static/styles/zulip.css

```diff
@@ -1620,6 +1619,7 @@ div.focused_table {
         margin-left: 1px;
         padding-right: 10px;
         margin-right: 1px;
+        overflow: visible;
```

Comment:
It seems weird to have the opposite explicitly set above in the group block and then overridden here; maybe we should set the `overlow: hidden` not on `.sub_count` at all?

Reply:
without overflow:hidden, we can get cases like this:
![image](https://user-images.githubusercontent.com/33805964/97955080-4fd72b00-1dcb-11eb-8f16-fb92e14dc41d.png)
without overflow:visible, the vertical bars disappear:
![image](https://user-images.githubusercontent.com/33805964/97955255-c1af7480-1dcb-11eb-8b5f-e329632285ea.png)